dvc/dvc.yaml

```diff
@@ -111,26 +111,6 @@ stages:
       outs:
       - ${item.results_file}
       # - ${item.output_wildcard}
-  explain-models:
-    foreach: ${explain_models}
-    do:
-      cmd: >-
-        morphoclass -v explain-models
-        --checkpoint-path ${item.checkpoint_path}
-        --results-file ${item.results_file}
-      deps:
-      - ${item.checkpoint_path}
-      outs:
-      - ${item.checkpoint_path_outs}
-  performance-report:
```

Comment:
Removed because the functionalities are now implemented in the following CLI sub-commands of `morphoclass`:
* `performance-table`
* `evaluate performance` — this is also available in the DVC stages `evaluate-***`
* `evaluate latent-features`
* `evaluate outliers`